dds/DCPS/RTPS/Spdp.cpp

```diff
@@ -4457,6 +4458,7 @@ void Spdp::SpdpTransport::thread_status_task(const DCPS::MonotonicTimePoint& /*n
         assign(data.participant_guid, guid);
         data.thread_id = i->first.c_str();
         data.utilization = 0.0;
+        data.saturated = false;
```

Comment:
Why is `saturated` necessary?  Can't the user determine their own limits?

Reply:
I added saturated to allow the relay to declare that its processing load is exceeding its load limit. Otherwise we are imposing on all the connecting apps to provide the configuration necessary to determine that the relay is busy and should therefore take a break before sending anything more messages.

Reply:
The only thing that needs to be available for the relay to do its jobs in the utilization of the discovery thread for the application participant.  The thread utilization of the rtps relay handler is not a concern at this time.